doc/api/test.md

```diff
@@ -920,6 +920,61 @@ test('runs timers as setTime passes ticks', (context) => {
 });
 ```
 
+## Snapshot testing
+
+> Stability: 1.0 - Early development
+
+Snapshot tests allow arbitrary values to be serialized into string values and
+compared against a set of known good values. The known good values are known as
+snapshots, and are stored in a snapshot file. Snapshot files are managed by the
+test runner, but are designed to be human readable to aid in debugging. Snapshot
+files should be checked into source control along with your test files. In order
+to enable snapshot testing, Node.js must be started with the
+[`--experimental-test-snapshots`][] command-line flag.
+
+Snapshot files are generated by starting Node.js with the
+[`--test-update-snapshots`][] command-line flag. A separate snapshot file is
+generated for each test entry point. By default, the snapshot file has the same
```

Comment:
So this is intended to be the main module for the current process. So if you run `node --test` and it runs `foo.test.js` and `bar.test.js`, those two files would be the entry points for their respective processes. I'm open to alternative wording here.

Reply:
I'm good with that. The only catch is if you have two test files `foo.test.js` and `bar.test.js` and those files import any other files that define tests (for example `common.test.js`). There would be no file created for `common.test.js` because the two processes would clobber each others files.

Reply:
IMO, `test files` is more clear.

Reply:
SGTM then.